src/uu/ps/src/picker.rs

```diff
@@ -54,13 +53,19 @@ fn time(proc_info: RefCell<ProcessInformation>) -> String {
     let cumulative_cpu_time = {
         let utime = proc_info.borrow_mut().stat()[13].parse::<i64>().unwrap();
         let stime = proc_info.borrow_mut().stat()[14].parse::<i64>().unwrap();
-        utime + stime
+        (utime + stime) / 100
     };
 
-    DateTime::from_timestamp_millis(cumulative_cpu_time)
-        .unwrap()
-        .format("%H:%M:%S")
-        .to_string()
+    let days = cumulative_cpu_time / (3600 * 24);
+    let hours = (cumulative_cpu_time % (3600 * 24)) / 3600;
+    let minutes = (cumulative_cpu_time % 3600) / 60;
+    let seconds = cumulative_cpu_time % 60;
+
+    if days != 0 {
+        format!("{:02}-{:02}:{:02}:{:02}", days, hours, minutes, seconds)
+    } else {
+        format!("{:02}:{:02}:{:02}", hours, minutes, seconds)
+    }
```

Comment:
I think you can move those to another function so that you can improve the testing coverage :P